hnet/objects.go

```diff
@@ -199,3 +205,40 @@ func NewStatsRequest() *StatsRequest {
 		UserIds: make([]uint32, 0),
 	}
 }
+
+type StatsResponse struct {
+	Stats  *UserStats
+	Status *Status
+}
+
+func (response StatsResponse) String() string {
+	return fmt.Sprintf(
+		"StatsResponse{Stats: %v, Status: %v}",
+		response.Stats,
+		response.Status,
+	)
+}
+
+func NewStatsResponse() *StatsResponse {
+	return &StatsResponse{
+		Stats:  NewUserStats(),
+		Status: NewStatus(),
+	}
+}
```

Comment:
On second thought I wonder if it would be better to move the `Status` struct into the `Stats` struct. What do you think?

Reply:
Alright, well. I just did it myself :D